docs/guide/start/concept-quickstart.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Brooklyn Concepts Quickstart
+layout: website-normal
+menu_parent: index.md
+---
+
+The following section gives a quick summary of the main Brooklyn concepts you will encounter in Getting Started.  For later investigation of these concepts see [The Theory Behind Brookln]({{site.path.website}}/learnmore/theory.html), and the detailed description in [Brooklyn Concepts]({{site.path.guide}}/concepts/).
+
+Having examined the concepts below, get started by **[installing and launching](running.html)** Brooklyn.
+
+
+***Deployment and Management*** Brooklyn is built for agile deployment of applications across cloud and other targets, and real-time autonomic management. "Autonomics" is the concept of components looking after themselves where possible (self-healing, self-optimizing, etc).
```

Comment:
"Autonomic Computing" I think is the official recognised term (it's the name of the Wikipedia page at least!)